include/firrtl.xml

```diff
@@ -128,6 +132,10 @@
       <context name="info" attribute="Info" lineEndContext="#stay">
         <DetectChar char="]" context="#pop"/>
       </context>
+      <context name="typemodifier" attribute="Keyword" lineEndContext="#pop">
+        <keyword String="types" context="#pop!type" attribute="Keyword"/>
+        <DetectChar char="{" context="#pop!field" attribute="Separator"/>
+      </context>
```

Comment:
This state transition seems a bit odd as it is dually encoding that `types` -> `types` and `{` -> `field`. Is it simpler to do something like popping on any whitespace that shows up?

This could probably work just like existing keywords which is just a `#stay`.

WDYT?

Reply:
Just using `#stay` in the default and field contexts should do the trick for valid FIRRTL, indeed.  This is a little more specific, but probably doesn't help us any, so let's go with simple.  Thanks!

Reply:
(updated accordingly)